Sources/NIO/Thread.swift

```diff
@@ -127,27 +127,33 @@ final class NIOThread {
 ///
 /// `ThreadSpecificVariable` is thread-safe so it can be used with multiple threads at the same time but the value
 /// returned by `currentValue` is defined per thread.
-///
-/// - note: `ThreadSpecificVariable` has reference semantics.
-public struct ThreadSpecificVariable<T: AnyObject> {
+public final class ThreadSpecificVariable<T: AnyObject> {
```

Comment:
Nit: while we're here, shall we update the generic type name? I think `Wrapped` is the Swifty name here.

Reply:
oh yes!

Reply:
went for `Value` because we're not really wrapping the value. But I wouldn't mind `Wrapped` either

Reply:
@Lukasa ^